docs/notes.rst

```diff
@@ -16,8 +21,12 @@ It also adds the following bug fixes and improvements:
 * Fix for setting *nullptr* in a *fixed_string*
 * Fix for v1.8.1 not building with -DBUILD_JAVA=ON
 * Fix for GAP messages not being sent in some cases
+* Fix for local/remote locators management
+* Fix for coverity report
+* Several memory issues fixes
 * `fastrtps.repos` file was updated
 * Documentation for building with Colcon was added
+* Change CMake configuration directory if INSTALLER_PLATFORM is set.
 
 **Note:** If you are upgrading from a version older than 1.7.0, it is **required** to regenerate generated source
 from IDL files using *fastrtpsgen*
```

Comment:
Probably because this was written early in the morning yesterday, it is missing https://github.com/eProsima/Fast-RTPS/commit/daf8b3bc1fc31084c7d8995152b60b1fe2ff5b53. That includes an optimization in the writer liveliness protocol when removing local writers, and a fix for a race condition when the built-in protocols were being created